### PR TITLE
Prevent Maintenance Mode in test environments

### PR DESCRIPTION
**Pull request by jasonbahl**
What does this implement/fix? Explain your changes.
---------------------------------------------------

This attempts to prevent WordPress from going into maintenance mode when tests are running.

### CODE REVIEW

**Comment by chriszarate** [comment]
Can we remove all the `sleep`s in functional tests?

https://github.com/wp-graphql/wp-graphql/blob/efaa5de3ef5a0c28d0b4ed28f92339ead159a2dc/tests/functional/ApolloPreflightRequestCept.php#L6

File: tests/_bootstrap/bootstrap.php [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <?php
+
+// Disable updates to prevent WP from going into maintenance mode while tests run
+define( 'WP_AUTO_UPDATE_CORE', false );
+add_filter( 'file_mod_allowed', '__return_false' );
```
Comment (jasonbahl) [inline tests/_bootstrap/bootstrap.php:5]:
```suggestion
```

**Review by jasonbahl — COMMENTED** [review]
Remove the filter

File: tests/_bootstrap/bootstrap.php [hunk 2]
```diff
@@ -1,4 +1,9 @@
 <?php
+
+// Disable updates to prevent WP from going into maintenance mode while tests run
+define( 'WP_AUTO_UPDATE_CORE', false );
+
```
Comment (jasonbahl) [inline tests/_bootstrap/bootstrap.php:5]:
```suggestion
add_filter( ‘enable_maintenance_mode’, ‘__return_false’ );
```

**Comment by jasonbahl** [comment]
> Can we remove all the `sleep`s in functional tests?
> 
> https://github.com/wp-graphql/wp-graphql/blob/efaa5de3ef5a0c28d0b4ed28f92339ead159a2dc/tests/functional/ApolloPreflightRequestCept.php#L6

@chriszarate yes we can, if I can get all the tests to pass without it! 🤞🏻

**Pull request merged**